cmd/server/main.go

```diff
@@ -94,6 +94,15 @@ func main() {
 	}()
 
 	go func() {
+		// Server should unlink the socket file prior to binding it in case the socket isn't cleaned up gracefully.
+		// This can happen if the application is killed by SIGKILL or SIGSTOP, i.e. kill -9 or docker kill by default.
+		zap.L().Info("unlinking ", zap.String("port", *addr))
```

Comment:
A couple changes:
* Can you make this message more informative? Right now, it will display `"unlinking "` every time the server is started without any explanation of what that means.
* Can you only remove the leftover socket if it exists? 
* Can you change the key name to "address", as you're referencing the socket path instead of "port"

```suggestion
		zap.L().Info("Removing existing socket", zap.String("address", *addr))
```

Reply:
have updated accordingly